### DISCUSSION TITLE
Exception from Rollup class

### DISCUSSION THREAD

**Discussion by nsinghCloudera** [jamessimone/apex-rollup#532] [Q&A] [answered]
Hi,
From last few weeks we are getting exception for the scheduled job of rollup summary, and I believe because of this our rollups are not working. 

**Apex script unhandled exception by user/organization: 0052H00000CT59X/00D90000000b5SO

Scheduled job 'Escalation Rollups' threw unhandled exception.

caused by: System.SerializationException: Tried to assign object of type System.AccessLevel to field of type RollupRepository.PermissionLevel in class: RollupAsyncProcessor.QueueableProcessor, while deserializing**

The user is system admin in the Org.

**Comment by jamessimone** [comment]
@nsinghCloudera thanks for reporting this. I should be able to fix this next week.

**Comment by nsinghCloudera** [comment]
@jamessimone I was wondering if there has been any progress in resolving this issue. Is there anything I can do to fix this issue or log it from my side?

**Reply by jamessimone** [reply]
Apologies - I lost track of this one because it was logged as a discussion. I'll take a look today.

**Reply by jamessimone** [reply]
@nsinghCloudera what version are you getting this error on?

**Reply by nsinghCloudera** [reply]
I'm on v1.5.83

**Reply by jamessimone** [reply]
I would try upgrading. I'll be making a change on 1.6.4 that may help, though.

**Reply by jamessimone** [reply, marked as answer]
@nsinghCloudera that update has now been released
